obspy/io/cybershake/tests/test_core_cybershake.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+"""
+The cybershake.core test suite.
+"""
+import numpy as np
+from obspy import read
+from obspy.io.cybershake.core import _is_cybershake, _read_cybershake
+
+
+class TestCybershake():
+    """
+    Testing reading of CyberShake seismograms.
+    """
+    def test_read_cybershake(self, testdata):
+        valid_file = testdata['test.grm']
+        invalid_file = testdata['not_cybershake.grm']
+        # Check that two files are/are not valid CyberShake data
+        assert _is_cybershake(valid_file)
+        assert not _is_cybershake(invalid_file)
+
```

Comment:
please separate the test function into two here (`test_is_cybershake()` and `test_read_cybershake()`) for more granularity in tests, testing "is()" and "read()" is two separate functionalities.